### ISSUE TITLE
id3 0.80 changes console codepage from OEM to ANSI

### ISSUE THREAD

**Issue by WonderRat** [squell/id3#5]
Windows XP SP3

```
>chcp
Текущая кодовая страница: 866

>id3.exe -V
id3 0.80 (2015356), Copyright (C) 2003, 04, 05, 06, 15 Marc R. Schoolderman
This program comes with ABSOLUTELY NO WARRANTY.

This is free software, and you are welcome to redistribute it
under certain conditions; see the file named COPYING in the
source distribution for details.

>chcp
╥хъє∙р  ъюфютр  ёЄЁрэшЎр: 1251
```

**Comment by squell** [comment]
Confirmed. This is caused by exit() bypassing a destructor which should return the codepage to its previous value.

**Comment by squell** [comment]
Fixed in commit 53c538f4db309b6907793e0e68f277a7d6a046f4
